src/backend/vulkan/src/lib.rs

```diff
@@ -701,64 +902,93 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             return Err(DeviceCreationError::MissingFeature);
         }
 
+        let extensions_registry = ExtensionsRegistry::new(vec![
+            // https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_AMD_negative_viewport_height.html
+            Extension::new(vk::AmdNegativeViewportHeightFn::name(), Version::V1_0)
+                .promoted_to(Version::V1_1),
+            // https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_EXT_descriptor_indexing.html
+            // Depends on `VK_KHR_get_physical_device_properties2`, which is an instance extension
+            Extension::new(vk::ExtDescriptorIndexingFn::name(), Version::V1_0)
+                .with_dependencies(vec![vk::KhrMaintenance3Fn::name()])
+                .promoted_to(Version::V1_2),
+            // https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_EXT_sampler_filter_minmax.html
+            // Depends on `VK_KHR_get_physical_device_properties2`, which is an instance extension
+            Extension::new(vk::ExtSamplerFilterMinmaxFn::name(), Version::V1_0)
```

Comment:
it sounds to me that all this information is supposed to be a part of the spec, so it could be upstreamed to `Ash`. Would you want to file a small issue to them to check if they are interested?

Reply:
I'll file an issue and resolve this with a link once I do

Reply:
https://github.com/MaikKlein/ash/issues/367